### PR TITLE
[terra-alert] alert focus shift to empty element fix

### PR DESCRIPTION
**Pull request by adoroshk**
**What was changed:**
This PR fixes the terra-alert focus shift change blocking terra-alert content from being interact-able.
The PR for the previous change: https://github.com/cerner/terra-core/pull/3885

### Testing
This change was tested using:

- [X] WDIO
- [X] Jest
- [X] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

A screenshot demonstrating the Show More button being accessible (focused):
Screenshot 2023-08-21 at 3 15 55 PM

Tested with VO and JAWS to ensure the change hasn't changed the assistive technologies announcements.

### Reviews

In addition to engineering reviews, this PR needs:
- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

** This PR is a follow-up PR for **

UXPLATFORM-9144

### CODE REVIEW

File: packages/terra-alert/tests/wdio/alert-spec.js [hunk 1]
```diff
@@ -72,6 +72,12 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
   });
 
   describe('Alert Long Text', () => {
+    it('focuses on show more button', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/alert/long-text-alert');
+      browser.keys(['Tab']);
+      Terra.validates.element('show more button focused');
```
Comment (sdadn) [inline packages/terra-alert/tests/wdio/alert-spec.js:78]: we should add a programmatic test for focus.

File: packages/terra-alert/src/Alert.module.scss [hunk 2]
```diff
@@ -33,6 +33,16 @@
     margin-top: var(--terra-alert-icon-margin-top);
   }
 
+  .content-container {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    line-height: 1.2;
```
Comment (cm9361) [inline packages/terra-alert/src/Alert.module.scss:41]: Should we be specifying a line height or let it be inherited.  Also, shouldn't the line height have units if it is required?

Comment (adoroshk) [reply packages/terra-alert/src/Alert.module.scss:41]: This styling was there before the change, I just changed an HTML layout a bit hence had to move some styling around per new class names, so it looks like a new code here, but it has been there a long time and it's out of the scope of this JIRA to change or keep it.

Comment (adoroshk) [reply packages/terra-alert/src/Alert.module.scss:41]: Here is the source of this styling: https://github.com/cerner/terra-core/blob/5f0141ba64d5ae282d5321629e7fdb0385884b18/packages/terra-alert/src/Alert.module.scss#L41

Comment (cm9361) [reply packages/terra-alert/src/Alert.module.scss:41]: ok.  It just looked new based on the changes.

File: packages/terra-alert/tests/wdio/alert-spec.js (hunk 1 above)
Comment (adoroshk) [reply packages/terra-alert/tests/wdio/alert-spec.js:79]: There is update to the tests: https://github.com/cerner/terra-core/pull/3889/commits/fcfbb3334033048c261b10bc89941bc6578a132d
Is that it, or need some other improvements? If more tests needed, the examples would be very appreciated.

**Review by nikhitasharma — APPROVED** [review]

File: packages/terra-alert/tests/wdio/alert-spec.js [hunk 3]
```diff
@@ -72,6 +72,22 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
...
+    });
+
+    it('open show hide upon button click', () => {
+      $('button.button').click();
+      Terra.validates.element('disclosed show hide');
+    });
+
+    it('close show hide', () => {
+      $('button.button').click();
```
Comment (trandrew1023) [inline packages/terra-alert/tests/wdio/alert-spec.js:88]: I think validation that the show hide was closed could be added here as well

Comment (adoroshk) [reply packages/terra-alert/tests/wdio/alert-spec.js:88]: Added here: https://github.com/cerner/terra-core/pull/3889/commits/20269f6fbdba9a65a3f4a2ad03501a2a069a200b

File: packages/terra-alert/tests/wdio/alert-spec.js (hunk 1 above)
Comment (sdadn) [reply packages/terra-alert/tests/wdio/alert-spec.js:79]: That looks good!

**Review by sdadn — APPROVED** [review]

**Review by kenk2 — APPROVED** [review]

**Pull request merged**